tests/pr_test.go

```diff
@@ -89,6 +72,24 @@ func TestRunCompleteUpgradeExample(t *testing.T) {
 	}
 }
 
+func setupFSCloudOptions(t *testing.T, prefix string) *testhelper.TestOptions {
+	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
+		Testing:       t,
+		TerraformDir:  fsCloudExampleTerraformDir,
+		Prefix:        prefix,
+		ResourceGroup: resourceGroup,
+		Region:        region,
+		TerraformVars: map[string]interface{}{
+			"skip_iam_authorization_policy": true, // The test account already has got a s2s policy setup that would clash
```

Comment:
Most of the time this auth policy will not exist actually, and this test will fail. The nuke nukes all global auth policies. We could add a feature to the nuke maybe to pass an ignore list of auth policies? Since policicies don't have a name, we can't use the `geretain` prefix to tell nuke to ignore them - we would have to add a new way to ignore policies.

Reply:
I actually think this is a good feature to have with the nuke, so I have created an internal issue for it

Reply:
We need to really review our testing approach here, I am spending 10x the time to get that pipeline to pass than actually coding the change. We have a problem.

Reply:
I'll comment it out, I think we need to invest in fresh account created for tests that touch account level resources like s2s policies. Once we have an enterprise account, we can create subaccounts for it for each new test run.

Reply:
FYI, we have the enterprise account already, so we should deep dive on next steps

Reply:
Let's prioritize this in next sprint.